src/Jupyter/Visualization/DisplayableExceptionEncoders.cs

```diff
@@ -107,11 +125,11 @@ public class DisplayableExceptionToTextEncoder : IResultEncoder
                 var builder = new StringBuilder();
                 builder.AppendLine(ex.Header);
                 var first = true;
-                foreach (var frame in ex.StackTrace ?? Enumerable.Empty<StackFrame>())
+                foreach (var frame in ex.StackTrace ?? Enumerable.Empty<DisplayableStackFrame>())
```

Comment:
In the code above ex.StackTrace is accessed without the null-coalescing operator (??). Do we need it here?

Reply:
No, good point, the assumptions should be the same.